### PR TITLE
Populates linked dataset and attributes

### PR DESCRIPTION
**Pull request by brichet**
This PR populates the *created links* panel (left) and the *identity links* (right).
It also display the actual name of the attributes, the ones in the data file and not the renamed ones in session file.

Code changes
-------------------

- adds *attributes* in the `sharedModel`
- adds a model in the link editor to compute the component links only once

Peek 2023-04-27 10-24

### CODE REVIEW

File: src/linkPanel/widgets/identityLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * @param clickCallback - Function to call when clicking on the delete icon.
+ */
+export function identityLinks(
+  links: [IComponentLinkInfo, boolean][],
+  clickCallback: (link: IComponentLinkInfo) => void
+): JSX.Element {
+  return (
+    <table>
+      {links.map(link => (
+        <tr>
```
Comment (trungleduc) [inline src/linkPanel/widgets/identityLinks.tsx:18]: Do we need the key props for the `tr` element?

File: src/schemas/data/attributes.schema.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "type": "object",
+  "description": "Attributes",
+  "title": "IAttributes",
+  "required": ["_type", "label"],
+  "additionalProperties": false,
+  "properties": {
+    "_type": {
```
Comment (trungleduc) [inline src/schemas/data/attributes.schema.json:8]: Do we need `_type`?

File: src/linkPanel/widgets/identityLinks.tsx (hunk 1 above)
Comment (brichet) [reply src/linkPanel/widgets/identityLinks.tsx]: Thanks @trungleduc 
I added it, but strangely react was not complaining about it.

File: src/schemas/data/attributes.schema.json (hunk 2 above)
Comment (brichet) [reply src/schemas/data/attributes.schema.json:8]: Since there are several types of attributes, I would say we should keep them.

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks @brichet

**Pull request merged**